docs/sql-migration-guide.md

```diff
@@ -87,7 +87,7 @@ license: |
 
   - In Spark version 2.4, when a spark session is created via `cloneSession()`, the newly created spark session inherits its configuration from its parent `SparkContext` even though the same configuration may exist with a different value in its parent spark session. Since Spark 3.0, the configurations of a parent `SparkSession` have a higher precedence over the parent `SparkContext`. The old behavior can be restored by setting `spark.sql.legacy.sessionInitWithConfigDefaults` to `true`.
 
-  - Since Spark 3.0, parquet logical type `TIMESTAMP_MICROS` is used by default while saving `TIMESTAMP` columns. In Spark version 2.4 and earlier, `TIMESTAMP` columns are saved as `INT96` in parquet files. To set `INT96` to `spark.sql.parquet.outputTimestampType` restores the previous behavior.
+  - Since Spark 3.0, parquet logical type `TIMESTAMP_MICROS` is used by default while saving `TIMESTAMP` columns. In Spark version 2.4 and earlier, `TIMESTAMP` columns are saved as `INT96` in parquet files. Note that, some SQL systems such as Hive 1.x and Impala 2.x can only read `INT96` timestamps, you can set `spark.sql.parquet.outputTimestampType` as `INT96` to restore the previous behavior and keep interoperability.
```

Comment:
see https://issues.apache.org/jira/browse/IMPALA-5050 and https://issues.apache.org/jira/browse/HIVE-22224

Impala supports parquet logical timestamp since 3.2.0, and Hive doesn't support it even in 2.3 (not sure if it's supported or not now).

Reply:
Actually, there may be a problem: In `HiveExternalCatalog.createTable`, we claim a parquet table is hive-compatible but it's actually not if the timestamp column is written as parquet logical timestamp. @MaxGekk can you look into it?

Reply:
I've created a ticket for it: https://issues.apache.org/jira/browse/SPARK-30875